src/app/components/top-nav.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import { Separator } from "@/components/ui/separator";
+import {
+  Sheet,
+  SheetContent,
+  SheetHeader,
+  SheetTrigger,
+} from "@/components/ui/sheet";
+import { cn } from "@/lib/utils";
+import { Bars3Icon, HomeIcon } from "@heroicons/react/20/solid";
+import Link, { LinkProps } from "next/link";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+
+type MobileLinkProps = LinkProps & {
+  children: React.ReactNode;
+  className?: string;
+  onOpenChange: (open: boolean) => void;
+};
+function MobileLink({ onOpenChange, className, ...props }: MobileLinkProps) {
+  const pathname = usePathname();
+  const isActive = pathname === props.href;
+  return (
+    <Link
+      aria-current={isActive ? "page" : undefined}
+      className={cn(
+        "flex flex-row items-center w-full rounded px-4 py-2 aria-[current='page']:bg-gray-100",
```

Comment:
This is a small but neat detail. 

The [aria-current attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) is used to ensure that a11y tools can tell which link on the page refers to the current page. 

Here we set `aria-current='page'` when this link has the same href as the current url path. 

We then use `aria-[current='page']` in the tailwind css to style it. This means that in order for our links to show the proper styling we _have_ to make our a11y better. 

Win for everyone 🚀

Reply:
That's very cool! Thank you for the link. We like our birds accessible.